release-process.rst

```diff
@@ -256,21 +291,28 @@ is Dec 31, 2026, 24:00 UTC, even if the actual release date slips to Jan 9,
  Feature selection and development
 ***********************************
 
-RFCs have been introduced many years ago and have been proven as being an
-amazing way to avoid conflicts while providing a very good way to propose new
-things to php.net. New features or additions to the core should go through the
-RFC process. It has been done successfully (as the process went well, but the
-features were not necessary accepted) already for a dozen of new features or
-improvements.
-
-Features can use branch(es) if necessary, doing so will minimize the impact of
-other commits and changes on the development of a specific feature (or the other
-way 'round). The shorter release cycle also ensures that a given feature can get
-into the next release, as long as the RFC has been accepted.
-
-The change to what we have now is the voting process. It will not happen anymore
-on the mailing list but in the RFCs directly, for php.net members, in a public
-way.
+RFCs were introduced many years ago and have proven to be an effective way to
+avoid conflicts while providing a structured process for proposing changes to
+the PHP programming language. Most new features or core additions SHOULD go
+through the RFC process. However, some features MAY be exempt, as described
+below. The process has been used many times for proposing new features and
+improvements, even when some proposals were ultimately not accepted.
+
+New features MUST be implemented and proposed using a GitHub pull request.
+
+Internal API changes (those that do not affect the user-facing API), as well as
```

Comment:
Do you mean extension-facing APIs as well here?

Reply:
Yeah I added that explicitly to the patch version rules so it should be covered. I just added a paragraph about updating UPGRADING and UPGRADING.INTERNALS.

Reply:
I will mark this as resolved but if you think there should be further clarification, please add a new comment / unresolve it.